proxies/events-graphql-proxy/src/schema/index.ts

```diff
@@ -0,0 +1,11 @@
+import { buildSubgraphSchema } from '@apollo/subgraph';
+
+import resolvers from './resolvers';
+import typeDefs from './typeDefs';
+
+// use mergeSchemas to make codegen generate typing correctly
+// for some reason makeExecutableSchema didn't work in index.ts for codegen
+
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-ignore
+export default buildSubgraphSchema({ typeDefs, resolvers });
```

Comment:
This is currently not in use, but it could be! The `proxies/events-graphql-proxy/src/index.ts` does not import this.

Reply:
I don't think the tsignore is needed anymore

Reply:
fixed both